pkg/report/output/summary/summary.go

```diff
@@ -170,7 +176,18 @@ func BuildReportString(rules map[string]*settings.Rule, results map[string][]Res
 		}
 	}
 
-	writeSummaryToString(reportStr, results, len(rules), failures, severityForFailure)
+	if reportPassed {
+		reportStr.WriteString("\nNeed to add your own custom rule? Check out the guide: https://curio.sh/guides/custom-rule\n")
+	}
+
+	emptySumary := writeSummaryToString(reportStr, results, len(rules), failures, severityForFailure)
+
+	if emptySumary {
+		writeEmptySuccessToString(len(rules), reportStr)
+		writeStatsToString(reportStr, config, lineOfCodeOutput, dataflow)
+	}
```

Comment:
Feels a bit weird to me that `writeSummaryToString` only writes something in the failure and warning cases, and we then handle the success summary separately. 

I'd suggest either renaming the `writeSummaryToString` method so that it's clearer, or moving the `writeEmptySuccessToString` call inside `writeSummaryToString` so the method still does what it claims to.